db/deploy/title_uniqueness.sql

```diff
@@ -0,0 +1,8 @@
+-- Deploy nasa-apt:title_uniqueness to pg
+-- requires: remove_reviewer_info_in_atbd_version
+
+BEGIN;
+
+ALTER TABLE apt.atbds ADD CONSTRAINT unique_title UNIQUE (title);
```

Comment:
Applying this migration will fail if we have existing titles that are not unique.

Reply:
Should we check this in dev and prod databases first?